packages/evolution-frontend/src/components/inputs/InputString.tsx

```diff
@@ -47,12 +47,14 @@ export const InputString = <CustomSurvey, CustomHousehold, CustomHome, CustomPer
             style={{ textTransform: props.widgetConfig.textTransform || ('none' as any) }}
             autoComplete="none"
             type="text"
+            pattern={props.widgetConfig.numericKeyboard ? "[0-9]*" : ""}
```

Comment:
How does this one work? I still can enter non numeric characters in  the car number input.